tests/unix_listener.rs

```diff
@@ -1,8 +1,11 @@
-#![cfg(all(unix, feature = "os-poll", feature = "net"))]
+#![cfg(all(feature = "os-poll", feature = "net", any(unix, feature = "os-ext")))]
 
+#[cfg(windows)]
+use mio::net;
```

Comment:
I think we need this because we need blocking version of the UDS types right? If so, let's put that in a separate module with that clearly documented.

Reply:
oh I interpreted your comment [here](https://github.com/tokio-rs/mio/pull/1610#discussion_r951790084) to mean we don't want to expose the blocking `std::os::unix::net` Windows equivalent types outside of the crate. 

If you'd like me to expose these for use in tests, where would be the best place for them to live?